lib/Test/Nginx/Socket.pm

```diff
@@ -2686,6 +2707,27 @@ will produce the following line (to C<stderr>) while running this test block:
 You need to remember to set the C<TEST_NGINX_NO_CLEAN> environment to 1 to prevent the nginx
 and other processes from quitting automatically upon test exits.
 
+=head2 response_body_filters
+
+Transform value for the body of the submitted request.
```

Comment:
This is ambiguous. It can be interpreted as "request body filters" instead of responses'. Better be more explicit here. Also, we need to emphasize that it's for the actual response body, instead of the expected one.